mysql.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+data:
+  mysql-root-password: TVlTUUxfUk9PVF9QQVNTV09SRA==
```

Comment:
The password is just `MYSQL_ROOT_PASSWORD`. Maybe using the secret is overkill but I didn't want to use a blank password.